Assets/AppsFlyer/AppsFlyerAdRevenue.cs

```diff
@@ -64,8 +91,61 @@ public static void setIsDebug(bool isDebug)
 
     }
 
+    public static integer setMediationNetworkType(AppsFlyerAdRevenueMediationNetworkType mediationNetwork)
```

Comment:
Maybe for better clarity, I would add some description of what do we have in this mapping. Or we can leave TODO, to align Android and iOS.

Reply:
what about `setMediationNetworkTypeAndroid`?